client/app/containers/test-details/test-details.html

```diff
@@ -137,17 +137,26 @@
                                             <p name="testName" class="test-info__name" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'black_font'">{{::test.name}}
                                                 <span data-ng-if="test.blocker" class="badge ng-binding bug-label-bg">BLOCKER</span></p>
                                             <div class="test-info__wrapper">
-                                                <span data-ng-if="test.finishTime && (test.finishTime - test.startTime)/1000 > 0 && test.status !== 'QUEUED'" data-ng-class="test.status === 'QUEUED' ? 'grey_font' : 'light_text'">
+                                                <span ng-if="test.finishTime && (test.finishTime - test.startTime)/1000 > 0 && test.status !== 'QUEUED'"
```

Comment:
You can add spaces before and after `/` ;)

Reply:
fixed